--- INSTALL.sh	2017-08-10 15:25:55.000000000 -0700
+++ newINSTALL.sh	2017-08-10 15:24:45.000000000 -0700
@@ -1,5 +1,8 @@
 #!/bin/bash
 #
+# This file was modified so that it doesn't use virtualenv
+# 
+# 
 # Copyright 2017 Google Inc.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
@@ -72,10 +75,12 @@
 which "$PYTHON"
 
 # Create a virtual environment for the pyclif installation.
-
 CLIF_VIRTUALENV="$INSTALL_DIR"/clif
-CLIF_PIP="$CLIF_VIRTUALENV/bin/pip"
-virtualenv -p "$PYTHON" "$CLIF_VIRTUALENV"
+
+# CLIF_PIP="$CLIF_VIRTUALENV/bin/pip"
+CLIF_PIP="pip"
+
+# virtualenv -p "$PYTHON" "$CLIF_VIRTUALENV"
 # Older pip and setuptools can fail.
 # TODO(gps): These work the first time but run into errors if rerun? ugh.
 # Regardless, *necessary* on systems with older pip and setuptools.  comment
@@ -112,4 +117,4 @@
 cp "$BUILD_DIR/tools/clif/protos/ast_pb2.py" clif/protos/
 "$CLIF_PIP" install .
 
-echo "SUCCESS - To use pyclif, run $CLIF_VIRTUALENV/bin/pyclif."
+echo "SUCCESS - To use pyclif, run $CLIF_VIRTUALENV/bin/pyclif."
\ No newline at end of file
